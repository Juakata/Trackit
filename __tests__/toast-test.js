import React from 'react';
import renderer from 'react-test-renderer';

import Toast from '../src/components/toast';

it('RoundButton snapShot', () => {
  const snap = renderer.create(
    <Toast visible text="Testing" />,
  ).toJSON();
  expect(snap).toMatchSnapshot();
});
