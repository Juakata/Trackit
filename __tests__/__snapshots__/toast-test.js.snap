// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoundButton snapShot 1`] = `null`;
